README.md

```diff
@@ -97,15 +97,17 @@ If you would like to scan for Illumina output, please see [runscanner-illumina/R
 <a id="debugging" />
 
 ## Debugging
+
+### Retrieving run output
 For troublesome runs, you can see the output for a particular run directory using:
 
     java -cp $RUN_SCANNER_HOME/WEB-INF/classes:$RUN_SCANNER_HOME/WEB-INF/lib/'*' ca.on.oicr.gsi.runscanner.scanner.Main
 
 It will display instructions on how to use it. You will have to set the `RUN_SCANNER_HOME` to the path containing an unpacked version of the WAR.
 
+### List observable runs
+To troubleshoot whether a processor can observe a run file, you can see a list of all the files a processor will accept from a directory using:
```

Comment:
Does it find all files in a directory, or all runs in a path?

Reply:
The files in the directory structure may not be of the right type (eg the PromethION processor only accepts .fast5 files, not .fastq). It lists all the files in the directory structure which the processor will process. (eg all the fast5 files, not listing any fastqs which may be present as well)

Reply:
For the PromethION processor, it will list all of the fast5 files within the specified directory (recursively), yes. For PromethION, a fast5 file is a run that it will process. For Illumina, it will list all of the subdirectories within the specified directory. For Illumina, the directory is the run. PacBio will be similar to Illumina. The only case for which it "lists all of the files that a processor will accept from a directory" is for PromethION, but in all cases, it will list all of the things that the processor will treat as runs.